core/core.ex

```diff
@@ -16,6 +16,8 @@ defmodule AntikytheraCore do
   """
   @impl true
   def start(_type, _args) do
+    Logger.add_translator({AntikytheraCore.ErlangLogTranslator, :translate})
+    Logger.configure(level: :info)
```

Comment:
It would be better to write a comment about why this setting is needed

Reply:
```elixir
Logger.add_translator({AntikytheraCore.ErlangLogTranslator, :translate})
# In dev or local environment, the log level is initially set to notice
# in order to avoid SASL progress reports.
# The log level is integrated into info after loading antikythera.
Logger.configure(level: :info)
```

How about this? Could you please tell me if you have any other ideas or better phrases.

Reply:
Looks good! But I'd like to change slightly.

```elixir
# In dev or local environment, the log level is initially set to `:notice` at mix_common.exs
# in order to avoid SASL progress reports.
# The log level is restored to `:info` after loading antikythera.
```

* Add `at mix_common.exs` to notify what file changed to `:notice`
    * Someone may want to see info level log during start up process to debug and find this `Logger.configure`.
* Change `integrated` to `restored`
    * IMHO, `integrated` is more difficult word than `restored`
* Use Elixir atoms, e.g. `:notice`, instead of English words

Reply:
Thank you for your advice!
Added at ab9bf44.